.gitignore

```diff
@@ -1,7 +1,5 @@
 .DS_Store
-.npmignore
 apps/*/translations/en
 node_modules
 public
-acceptance_tests/output/
```

Comment:
maybe... I haven't really looked at it but it doesn't seem that output is currently being output anywhere